examples/acquiring_data/downloading_cutouts.py

```diff
@@ -23,15 +23,15 @@
 #####################################################
 # First, query a full frame AIA image.
 
-t0 = astropy.time.Time('2012-09-24T14:56:03', scale='utc', format='isot')
+start_time = Time('2022-01-01T00:00:00', scale='utc', format='isot')
```

Comment:
I am hoping that a more recent set of data might help with the failure of the download.

Reply:
I am 👎 on this change. As I said on the call yesterday, the date and coordinates of the cutout were chosen specifically to follow an active region observed on that date. This is often what people use the cutout service for. The example is less compelling with a random date/cutout.

Reply:
Fair, I will update the time and crop region to be an AR, would that be ok?

Reply:
I have updated the crop to follow an AR, it has some nice loops.

Reply:
Also should display now on the gallery page.